### DISCUSSION TITLE
connect deta database to esp32

### DISCUSSION THREAD

**Discussion by Hermann002** [deta/space-docs#100] [Q&A]
Hello, please how can i connect the deta database to ESP 32 ?
here is my code...

#include <detaBaseArduinoESP32.h>
#include <WiFiClientSecure.h>
#define LED 2

char* apiKey = "a0piahtg37g_As5KJrq9cyGRUjU6KV5CjvPy1mMD7Pni";
char* detaID = "0piahtg37g";
char* detaBaseName = "test_dataBase";

WiFiClientSecure client;
DetaBaseObject detaObj(client, detaID, detaBaseName, apiKey, true);

**Comment by lemonyte** [comment]
Please direct your question to the maintainer of the library you are using (https://github.com/A223D/detaBaseArduinoESP32).

Also generate a new data key and delete this one because anyone can now access your Deta Base.
